lambda/rotator-snowflake/setup/okta.go

```diff
@@ -0,0 +1,13 @@
+package setup
```

Comment:
at some point you might want to split these packages into "lambda/rotator-snowflake/pkg/okta" "lambda/rotator-snowflake/pkg/databricks" is my guess. probably fine for now though

Reply:
yeah, probably for another PR